quest-7/README.md

```diff
@@ -1,7 +1,41 @@
-# 💥 Quest 7 - Use Solana Pay to Defend Your Ship
+# Solana Pay -- Pirates Edition
```

Comment:
We should probably come up with a "Cthulu-themed" title and use it in the main README as well